### PR TITLE
GDEV-225 Create an ECR module

### PR DESCRIPTION
**Pull request by BenjaminPLeon**

### CODE REVIEW

File: aws/ecr/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  }
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# OUTPUTS
+# ----------------------------------------------------------------------------------------------------------------------
+
+output "ecr" {
+  value = aws_ecr_repository.repo
+}
```
Comment (TysonBeck) [inline aws/ecr/ecr.tf]: Need a new line at the end

Comment (BenjaminPLeon) [reply aws/ecr/ecr.tf]: Updated in https://github.com/gravicore/terraform-gravicore-modules/pull/229/commits/4dd505db5b00b8aa9e3250431f0a192b4fd281b9

**Review by TysonBeck — DISMISSED** [review]

**Review by jeremyodle — CHANGES_REQUESTED** [review]
- [x] Missing labeling `aws/ecr`

**Comment by BenjaminPLeon** [comment]
> * [x]  Missing labeling `aws/ecr`

Added in https://github.com/gravicore/terraform-gravicore-modules/pull/229/commits/83cc5407415060a5760af53143f08772343998c7

**Review by jeremyodle — APPROVED** [review]

**Pull request merged**